content/docs/Configuration/downstream/pull_from_upstream.md

```diff
@@ -0,0 +1,66 @@
+---
+title: Release from downstream
```

Comment:
Do we call it somewhere else like this?

Reply:
No, should I switch to _Propose downstream_ and _Pull from upstream_? though it feels a bit weird next to _Copr builds_, etc.

Reply:
I would try to stick with the job names. Both because of the consistency with the job name but also because we've already been using that name for some time.